tests/test.cpp

```diff
@@ -107,7 +107,7 @@ flatbuffers::DetachedBuffer CreateFlatBufferTest(std::string &buffer) {
   Test tests[] = { Test(10, 20), Test(30, 40) };
   auto testv = builder.CreateVectorOfStructs(tests, 2);
 
-  // clang-format off
+// clang-format off
```

Comment:
why were all of these moved?

Reply:
idk, I just ran clang-forma 🤷

Reply:
I ran git-clang-format against master and they were unmoved